orb.yml

```diff
@@ -11,19 +11,69 @@ commands:
   install:
     description: "Installs the nexus-platform-api and CLI script"
     parameters:
-      nexus-platform-api-version:
+      groovy-version:
         type: string
-        default: "3.3.20181025-095610.1ce86ad"
+        default: "2.5.3"
     steps:
     - run: |
-        curl http://central.maven.org/maven2/com/sonatype/nexus/nexus-platform-api/<< parameters.nexus-platform-api-version >>/nexus-platform-api-<< parameters.nexus-platform-api-version >>.jar -o nexus-platform-api.jar
+        curl -L https://dl.bintray.com/groovy/maven/apache-groovy-binary-<<parameters.groovy-version>>.zip -o apache-groovy-binary.zip
+        curl -L https://raw.githubusercontent.com/sonatype-nexus-community/circleci-nexus-orb/INT-1109_repository_groovy_wrapper/cli/src/main/groovy/NexusPublisher.groovy -o NexusPublisher.groovy
+        unzip apache-groovy-binary.zip
+        mv groovy-<< parameters.groovy-version>> groovy
 
   archive:
+    description: "Archives artifacts to Nexus Repository Manager"
+    parameters:
+      username:
+        type: string
+        default: "admin"
+      password:
+        type: string
+        default: "admin123"
+      serverurl:
+        type: string
+        default: "http://localhost:8081/"
+      filename:
+        type: string
+        default: "/tmp/workspace/test1-1.0-SNAPSHOT.jar"
+      format:
+        type: string
+        default: "maven2"
+      groupId:
+        type: string
+        default: "com.example"
+      artifactId:
+        type: string
+        default: "myapp"
+      version:
+        type: string
+        default: "1.0"
+      extension:
+        type: string
+        default: "jar"
+      repository:
+        type: string
+        default: "maven-releases"
+
     steps:
-    - run: echo "Archive command will start here"
+    - attach_workspace:
+        at: /tmp/workspace
+    - run: >
+        groovy/bin/groovy NexusPublisher.groovy
+        --username <<parameters.username>>
+        --password <<parameters.password>>
+        --serverurl <<parameters.serverurl>>
+        --filename <<parameters.filename>>
+        --format <<parameters.format>>
+        -CgroupId=<<parameters.groupId>>
+        -CartifactId=<<parameters.artifactId>>
+        -Cversion=<<parameters.version>>
+        -Aextension=<<parameters.extension>>
+        --repository <<parameters.repository>>
```

Comment:
I'm not a yaml expert, but the '>' notation at the beginning turns newlines into spaces and adds a newline at the very end.  See here:  https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines